himportscan/src/HImportScan/GHC/Settings.hs

```diff
@@ -0,0 +1,50 @@
+-- Copyright (c) 2020, Shayne Fletcher. All rights reserved.
+-- SPDX-License-Identifier: BSD-3-Clause.
+
+{-# OPTIONS_GHC -Wno-missing-fields #-}
+{-# OPTIONS_GHC -Wno-name-shadowing #-}
+
+-- This file is a single code path copied over from https://hackage.haskell.org/package/ghc-lib-parser-ex-8.10.0.24/docs/src/Language.Haskell.GhclibParserEx.GHC.Settings.Config.html
+-- TODO[GL]: We can get rid of this file once we only support >=9.2, as ParserOpts are much smaller there.
+module HImportScan.GHC.Settings(
+    fakeSettings
+  , fakeLlvmConfig
+  )
+where
```

Comment:
Another way to create some dynflags could be to use the `Ghc` monad.
```haskell
runGhc (Just libdir) $ do
      dflags <- getSessionDynFlags
      liftIO (getImports (tweakYourLanguageExtensions dflags) ...)
```
We would have to see if it has much of a startup overhead though.

Reply:
Oh right. I totally forgot we can actually run a ghc, I'm kind of in the "ghc-lib-parser mindset" :sweat_smile:. I guess it shouldn't be that bad if we only run one ghc session - we are not lightning fast anyway. I'll try and use this if there isn't a big noticeable difference.

Reply:
I did this. However this dragged in the `ghc-paths` dependency. I also tried doing `ghc --print-libdir` instead, but that didn't work. I guess we could potentially replicate exactly what `ghc-paths` is doing locally, so that we don't have that dependency, if you'd prefer.

Reply:
Thanks. `ghc-paths` is lean enough to keep, I think.

Reply:
No longer relevant, as we droppped the "run ghc" approach.